gen/calc.py

```diff
@@ -1003,6 +1003,7 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'dns_search': '',
         'auth_cookie_secure_flag': 'false',
         'marathon_java_args': '',
+        'marathon_port': '8080',
```

Comment:
This is under the `default` section, which means user config can override it, which isn't what we want. This line needs to go under `must`, which is down below.